src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,800;1,800&display=swap" rel="stylesheet" crossorigin="anonymous">
+    <title>Leaderboard</title>
+</head>
+<body>
+    <main class="main-section">
+        <h1 class="main-heading">Leaderboard</h1>
+
+        <section class="main-scores">
+            <div class="recent-scores">
+                <div class="recent-scores-other">
+                    <h2>Recent scores</h2>
+                    <button type="button" class="refresh">Refresh</button>
```

Comment:
- [ ] The refresh button does not work, when I click it nothing happens, kindly fix this 😄

Reply:
Hello @helman101  I have worked on the requested changes. Thanks!